docs/usage/8.md

```diff
@@ -0,0 +1,298 @@
+# Restricted Network Image Mirroring for OpenShift 4.2
+
+## Introduction
+This guide was develeped using the [generic instructions](https://docs.openshift.com/container-platform/4.2/operators/olm-restricted-networks.html) for mirroring operators from OLM. 
+
+The steps in these instructions roughly correspond with the steps in the document linked above.
+
+There are a few exports that if set make these commands easier to run. It is likely you'll only need to change the values for `QUAY_TOKEN` through `exposed_registry`. The remainder are for us supporting dev/stage/prod environments with this script more easily.
+
+## Step 0a: Update and Export environment variables
+
+### Prod:
+```
+export QUAY_TOKEN="basic ..."
+export local_app_registry=quay.io
+export local_app_ns=
+export local_registry=image-registry.openshift-image-registry.svc:5000
+export local_registry_ns=rhcam-1-0-restricted
+export exposed_registry=$local_registry
+export ocs_source_registry=registry.redhat.io
+export ocs_source_registry_ns=ocs4
+export source_registry=registry.redhat.io
+export source_registry_ns=rhcam-1-0
+export remote_registry=registry.redhat.io
+export remote_registry_ns=rhcam-1-0
+export remote_registry_prefix=""
+export remote_app_ns=redhat-operators
+export coperator_image=openshift-migration-rhel7-operator
+export roperator_image=rook-ceph-rhel8-operator
+export moperator_image=mcg-rhel8-operator
+export ooperator_image=ocs-rhel8-operator
+export noobaa_image=mcg-core-rhel8
+export controller_image=openshift-migration-controller-rhel8
+export ui_image=openshift-migration-ui-rhel8
+export velero_image=openshift-migration-velero-rhel8
+export plugin_image=openshift-migration-plugin-rhel8
+export helper_image=openshift-migration-velero-restic-restore-helper-rhel8
+```
+
+## Step 0b: To use the internal openshift registry you can expose and push to the internal registry.
+```
+git clone https://github.com/fusor/mig-operator
+pushd mig-operator/deploy/test-helpers
+cp my_var.dev.ex my_var
+./expose_cluster_registry.sh
+./docker_login_exposed_registry.sh
+popd
+oc new-project ${local_registry_ns}
+oc new-project ${source_registry_ns}
+oc policy add-role-to-group system:image-puller system:serviceaccounts:openshift-migration --namespace=${local_registry_ns}
+oc policy add-role-to-group system:image-puller system:serviceaccounts:openshift-migration --namespace=${source_registry_ns}
+```
+Update the exposed_registry export value
```

Comment:
Can we give a bash snippet of how to correctly update this value? Unclear where the value comes from just reading the doc.

Reply:
Step 0a.... "There are a few exports that if set make these commands easier to run. It is likely you'll only need to change the values for QUAY_TOKEN through exposed_registry. The remainder are for us supporting dev/stage/prod environments with this script more easily."